docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+---
+id: google-iap
+title: Google IAP
+sidebar_label: Google IAP
+---
+
+## Introduction
+
+As of September 2021, Subscription Platform (SubPlat) supports [Google IAP](https://developer.android.com/google/play/billing) integrations for products with Mozilla VPN as one example. An integration allows us to maintain an awareness of a Google IAP subscriber's subscription status for a given android app, including any state changes, which we forward on to the relevant relying party (RP).
+
+Importantly, Google provides read-only access to Google IAP subscriptions. Consequently, we do not (and cannot) modify Google IAP subscriptions (e.g. making plan changes such as upgrades).
+
+A more technical discussion (geared toward Firefox Accounts (FxA) engineers) can be found in [Google IAP Integration](/ecosystem-platform/tutorials/subscription-platform#google-iap-integration).
+
+## Prerequisites
+
+This guide assumes you are already integrated with FxA/SubPlat; please see [Integration with Subscription Platform](/ecosystem-platform/relying-parties/tutorials/integration-with-subscription-platform) for more information.
+
+Ideally, you will also want to grant 1-2 SubPlat engineers working on the integration developer access to your android app in [Google Play Console](https://play.google.com/console) ([example bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1796624)). This allows us to access your android app’s [Google Play Developer API](https://developers.google.com/android-publisher) keys, to look up needed app-specific information and to configure an endpoint to receive [Real-time developer notifications](https://developer.android.com/google/play/billing/rtdn-reference).
+
+## Configuration
+
+These steps can be done by a product manager or developer and in parallel with steps in [SubPlat API Calls](#subplat-api-calls). More information regarding specific Stripe metadata keys mentioned below can be found in the [Subscription Platform integration tutorial](/ecosystem-platform/tutorials/subscription-platform#stripe-productplans).
+
+### Create an IAP configuration document
+
+This document represents a list of Play Store plans (identified uniquely by their Play Store [`token`](https://developer.android.com/google/play/billing#tokens) for your product.
```

Comment:
From my understanding the token is the unique identifier for Google when tied to a customer, and only changes when the subscription plan changes, but please let me know if that is incorrect and I will update.

Reply:
Fixed! `sku` was renamed to `productId` among other changes: https://developer.android.com/google/play/billing/compatibility#managing-subscriptions. After discussing with Ben, it was decided to leave `productId` in the docs as that is what RPs will see. I included a new section under `Introduction` to mention the changes and noted the backwards compatibility and how we use `sku` in our codebase.